config/application.rb

```diff
@@ -122,6 +122,9 @@ class Application < Rails::Application
       config.branding = config_for(:branding).deep_symbolize_keys
     end
 
+    # org abbreviation for the root google analytics tracker that gets planted on every page
+    config.tracker_root = "UoE"
```

Comment:
Probably want something more generic here like 'DMPRoadmap' so that other installations aren't using UoE by default.

Reply:
Agh! that wasn't intentional